### PR TITLE
Add support for publickey-credentials-create permission policy

### PR DESCRIPTION
**Pull request by stephenmcgruer**
This PR adds support for `publickey-credentials-create` permission policy,
to support https://github.com/w3c/webauthn/pull/1801

See also https://github.com/w3c/webauthn/issues/1656

***
<a href="https://pr-preview.s3.amazonaws.com/stephenmcgruer/webappsec-credential-management/pull/209.html" title="Last updated on Feb 27, 2023, 3:51 PM UTC (305097c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/209/976f91b...stephenmcgruer:305097c.html" title="Last updated on Feb 27, 2023, 3:51 PM UTC (305097c)">Diff</a>

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
(I believe the build fails because the corresponding WebAuthn PR is not landed, so the dfn reference isn't there yet)

**Comment by stephenmcgruer** [comment]
cc @ve7jtb , who I believe wished to make sure that this was compatible with the previous L2 discussion on what create() in a cross-origin iframe would look like (particularly the permission policy name, I believe).

File: index.bs
Comment (johannhof) [inline index.bs:1109]:
```suggestion
    5.  If |options|[{{CredentialCreationOptions/publicKey}}] [=map/exists=] and
```

(to be consistent with the wording in Request a Credential)

**Review by nsatragno — APPROVED** [review]
Looks good. Let's wait for the webauthn PR to be merged before merging this one.

**Comment by aaahlam621997** [comment]
> يضيف هذا PR دعما لسياسة الأذونات ، لدعم [w3c / webauthn # 1801](https://github.com/w3c/webauthn/pull/1801)`publickey-credentials-create`
> 
> انظر أيضا [w3c/webauthn#1656](https://github.com/w3c/webauthn/issues/1656)
> 
> [معاينة](https://pr-preview.s3.amazonaws.com/stephenmcgruer/webappsec-credential-management/pull/209.html) | [فرق](https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/209/976f91b...stephenmcgruer:305097c.html)

**Comment by stephenmcgruer** [comment]
(Closing/opening just to kick the CI into re-running)

**Comment by stephenmcgruer** [comment]
So the CI is broken because the navigation re-write in HTML (https://github.com/whatwg/html/pull/6315) un-exported the concept of a `parent browsing context`, and so the entire webappsec-credential-management spec will no longer build. Guidance from HTML folks is that the spec should be talking in terms of Navigables instead of browsing contexts, which I don't immediately understand very well 🤣 

This is not related to this PR, so my suggestion is that we ignore the failing CI here and open an issue to track the failing build - wdyt @nsatragno ?

**Comment by nsatragno** [comment]
SGTM

**Pull request merged**
